pretext/cli.py

```diff
@@ -405,13 +405,19 @@ def init(refresh: bool, files: List[str]) -> None:
     type=click.STRING,
     help="xml:id of the root of the subtree to be built.",
 )
+@click.option(
+    "--no-knowls",
+    is_flag=True,
+    help="Suppress knowl generation for html-format output (e.g. for faster previews of individual sections)",
```

Comment:
I reworded the help to make it clear that this is not for keeping content unknowled, but just for previews.

Reply:
It's not really "for faster previews of individual sections" though. It's for using hyperlinks rather than knowls (since the knowl files aren't generated either way for subtree biulds).

How about:

```suggestion
    help="Use hyperlinks instead of knowls (e.g. for previewing individual sections when knowl files from other sections may not exist)",
```

Reply:
That's great.  And this is good, because clearly I didn't not understand it until now.

Reply:
ah, you didn't commit my suggestion though?

This is a feature I've started using a lot, where instead of committing over other folks' PRs, you can use this button:

![image](https://github.com/PreTeXtBook/pretext-cli/assets/1559632/23323336-adce-4ae4-82f7-c3c11c3c8db8)

to suggest a change, and the other person can commit the change if they agree.

I'll make a hotfix PR to implement this.